pydantic/json_schema.py

```diff
@@ -1,5 +1,5 @@
-"""
-Usage docs: https://docs.pydantic.dev/2.5/concepts/json_schema/
```

Comment:
> Also, removes the burden of updating version numbers in docs links and ensures we're using markdown links instead of a mix of markdown ant http refs.

Indeed!